akvo/rsr/spa/app/modules/editor/section1/section1.jsx

```diff
@@ -70,6 +70,7 @@ const Info = ({ validations, fields, projectId, errors, showRequired, program, d
     }
   }, [fields.title])
   const disableMWCFields = fields?.program?.id === 9062
+  const projects = results && results.filter(it => it.id !== Number(projectId))
```

Comment:
I think you can remove results check and just add optional chaining

Reply:
noted @navins94 will do, Thank you